functions.sh

```diff
@@ -35,7 +35,7 @@ get_io_commits() {
 }
 
 get_patch_commits() {
-	find $PATCH_DIR -type f -printf %f\\n|cut -d. -f1
+	find $PATCH_DIR -type f|xargs -n 1 basename|cut -d. -f1
```

Comment:
It's probably not really necessary, but consider using `find -print0` and `xargs -0`.